SwiftSyntaxDevUtils/Sources/swift-syntax-dev-utils/common/ProcessRunner.swift

```diff
@@ -0,0 +1,117 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2018 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+
+/// Provides convenience APIs for launching and gathering output from a subprocess
+public class ProcessRunner {
+  private static let serialQueue = DispatchQueue(label: "\(ProcessRunner.self)")
+
+  private let process: Process
+
+  public init(
+    executableURL: URL,
+    arguments: [String],
+    additionalEnvironment: [String: String] = [:]
+  ) {
+    process = Process()
+    process.executableURL = executableURL
+    process.arguments = arguments
+    process.environment = additionalEnvironment.merging(ProcessInfo.processInfo.environment) { (current, _) in current }
```

Comment:
This prefers the value from `additionalEnvironment`, right? In that case, I think `current` should be named `additional`.